plans/podman_build_test.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -eox pipefail
+
+rpm -q golang
+
+if [ -f /etc/fedora-release ]; then
+    export TMPDIR=/var/tmp
+fi
+
+git clone https://github.com/containers/podman
```

Comment:
I think this can use `--depth 1` to not pull the entire history which should safe time.
Also IMO the directory structure makes no sense, podman as subdir from common is extremely confusing espacally in the go mod replace command. I think they should be side by side instead.

Reply:
This will be problematic and confusing to a developer that just made changes in their PR.  Those changes won't be reflected when this script runs (since it's pulling from `main`).  It will also similarly be irreverent if/when run on a release branch or for a tag.  I'm surprised the framework doesn't already clone the repository from the appropriate context, doesn't it?

Reply:
this is cloning podman but we are in common so we of course we have to get podman if we want to build test podman + c/common PR

Reply:
Oh durr, nevermind.  I failed to look at the PR URL :cry: